WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/WelcomePage.java

```diff
@@ -95,43 +97,47 @@ private Composite createQuestionComposite(final Composite parent) {
 		Image questionIconImage = questionIconImageDescriptor.createImage();
 		questionIcon.setImage(questionIconImage);
 
-		UIUtils.createLabel("   " + labelQuestion, composite);
+		UIUtils.createBoldLabel("   " + labelQuestion, composite);
 
-		final Composite radioButtons = UIUtils.createFullGridedComposite(
-				composite, 1);
-		radioButtons.setLayoutData(new GridData(SWT.BEGINNING, SWT.CENTER,
-				false, true));
-		radioButtons.setLayout(new FillLayout());
-		final Button radioButtonYes = UIUtils.createRadioButton(radioButtons,
-				"Yes");
+		radioButtonYes = UIUtils.createRadioButton(composite, "Yes");
 		radioButtonYes.addSelectionListener(new SelectionListener() {
 
 			@Override
 			public void widgetSelected(SelectionEvent e) {
+				setErrorMessageAndPageComplete(null);
 				removeDynamicContent(parent);
-				dynamicContent = createLoginComposite(parent);
-				setTitle(title + " (1/1)");
+				dynamicContent = createWelcomeComposite(parent);
+				setTitle(title
+						+ " ("
```

Comment:
By pulling this calculation of the page number up into the base class, you avoid a lot cloned and difficult to read code.